### ISSUE TITLE
[Signature does not match???] 

### ISSUE THREAD

**Issue by PolIonius** [pizzaboxer/bloxstrap#1008]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

I tried to install bloxstrap today and this issue popped up. I don't know what it means. 
![image](https://github.com/pizzaboxer/bloxstrap/assets/152827599/74f2a34e-d5e3-4e3e-9807-0cb776ee9a92)
[Bloxstrap_20231203T221352Z.log](https://github.com/pizzaboxer/bloxstrap/files/13541079/Bloxstrap_20231203T221352Z.log)

**Comment by PolIonius** [comment]
I actually havent, might do that and i"ll send the effect of it here.

**Comment by PolIonius** [comment]
Yeah I just reinstalled it after removing every file involved, same issue.
![image](https://github.com/pizzaboxer/bloxstrap/assets/152827599/66347888-79ea-4369-b232-edd1e4db1368)

**Comment by pizzaboxer** [comment]
Read the preliminary instructions next time. https://github.com/pizzaboxer/bloxstrap/wiki/%22Signature-does-not-match!%22-error-when-launching-Roblox

I'll reopen this if you need further help
